src/modules/peek/Peek.FilePreviewer/Previewers/ImagePreviewer/ImagePreviewer.cs

```diff
@@ -83,10 +84,18 @@ private Task LoadLowQualityThumbnailAsync()
 
                 if (!IsFullImageLoaded && !IsHighQualityThumbnailLoaded)
                 {
-                    // TODO: Handle thumbnail errors
-                    ThumbnailHelper.GetThumbnail(Path.GetFullPath(File.Path), out IntPtr hbitmap, ThumbnailHelper.LowQualityThumbnailSize);
-                    var thumbnailBitmap = await GetBitmapFromHBitmapAsync(hbitmap);
-                    Preview = thumbnailBitmap;
+                    if (File.Extension == ".png")
```

Comment:
I think we might want to introduce a `PngPreviewer` in the future instead of having conditional logic in here! Could be a separate pr.